css/properties/touch-action.json

```diff
@@ -0,0 +1,215 @@
+{
+  "css": {
+    "properties": {
+      "touch-action": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/touch-action",
+          "support": {
+            "webview_android": {
+              "version_added": "36"
+            },
+            "chrome": {
+              "version_added": "36"
+            },
+            "chrome_android": {
+              "version_added": "36"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "52",
+                "notes": "Not applicable to Firefox platforms that support neither pointer nor touch events."
+              },
+              {
+                "version_added": "29",
+                "version_removed": "52",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.touch_action.enabled",
+                  "value_to_set": "true"
+                },
+                "notes": "Not applicable to Firefox platforms that support neither pointer nor touch events."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "52"
+              },
+              {
+                "version_added": "29",
+                "version_removed": "52",
+                "flag": {
+                  "type": "preference",
+                  "name": "layout.css.touch_action.enabled",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": [
+              {
+                "version_added": "11",
+                "notes": "IE additionally supports the non-standard values <a href='https://msdn.microsoft.com/zh-cn/library/windows/apps/hh767313.aspx'>double-tap-zoom, cross-slide-x and cross-slide-y</a>."
```

Comment:
We should drop the note and list the non-standard values below (with only IE supporting them), like you did for other values.

Reply:
Afaik `cross-slide-x` and `cross-slide-y` don't actually do anything on the web, they're just values that will parse for metro apps. MSDN excludes them from its syntax definition but includes them in the list of values beneath. Should they still have rows?